cmake/onnxruntime_framework.cmake

```diff
@@ -25,7 +25,6 @@ endif()
 
 if (onnxruntime_MINIMAL_BUILD)
   set(onnxruntime_framework_src_exclude
-    "${ONNXRUNTIME_ROOT}/core/framework/provider_bridge_ort.cc"
```

Comment:
so we want to include it in minimal build?

Reply:
Ah, this is for framework, I added a similar cmake block in session to do this also.